deployment/sklearn/sklearn-census-income-prediction.ipynb

```diff
@@ -0,0 +1,629 @@
+      "metadata": {
+        "id": "Ad4pWiXaH16Z"
+      },
+      "outputs": [],
+      "source": [
+        "model.fit(X, y)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "wqq0jVXgH3O2"
+      },
+      "outputs": [],
+      "source": [
+        "train_acc = model.score(X, y)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "C_bYCiwRHxKU"
+      },
+      "outputs": [],
+      "source": [
+        "print(f\"Training accuracy: {round(train_acc, 4)}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "o3dUJdBjITaF"
+      },
+      "source": [
+        "## 5. Model Register"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VM0tJgXNxjLS"
+      },
+      "outputs": [],
+      "source": [
+        "with open('model.pkl', 'wb') as f:\n",
+        "    cloudpickle.dump(model, f)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "uHIKatKcsPsB"
+      },
+      "outputs": [],
+      "source": [
+        "class Predictor(VertaModelBase):\n",
+        "    def __init__(self, artifacts):\n",
+        "        self.model = cloudpickle.load(open(artifacts['serialized_model'], 'rb'))\n",
+        "        \n",
+        "    @verify_io\n",
+        "    def predict(self, data):\n",
+        "        results = []\n",
+        "\n",
+        "        for item in data:\n",
+        "            results.append(self.model.predict(item).tolist())\n",
+        "        \n",
+        "        return results\n",
+        "\n",
+        "    def describe(self):\n",
+        "        return {\n",
+        "            'method': 'predict',\n",
+        "            'args': 'age,workclass,fnlwgt,education,education-num,marital-status,occupation,relationship,race,sex,capital-gain,capital-loss,hours-per-week,native-country',\n",
+        "            'returns': 'income_label',\n",
+        "            'description': 'Predicts whether a person has >50k income based on census data.',\n",
+        "            'input_description': 'Batch of census information, one sample per entry.',\n",
+        "            'output_description': 'String indicating whether the person earns more than 50k a year.'\n",
+        "        }\n",
+        "        \n",
+        "    def example(self):\n",
+        "        return [[\n",
+        "            [49, 4, 160187, 6, 5, 3, 8, 1, 2, 0, 0, 0, 16, 23],\n",
+        "            [52, 6, 209642, 11, 9, 2, 4, 0, 4, 1, 0, 0, 45, 39],\n",
+        "            [31, 4, 45781, 12, 14, 4, 10, 1, 4, 0, 14084, 0, 50, 39],\n",
+        "            [42, 4, 159449, 9, 13, 2, 4, 0, 4, 1, 5178, 0, 40, 39],\n",
+        "            [37, 4, 280464, 15, 10, 2, 4, 0, 2, 1, 0, 0, 80, 39]\n",
+        "        ]]"
```

Comment:
fyi, I think some of our existing sklearn examples are inconsistent about this so it's a bit confusing, but this kind of has an unnecessary layer of `[]`. `predict()` could be written without the `for` loop, just passing `data` directly to `self.model.predict()`.